ci-operator/step-registry/hypershift/mce/power/create/nodepool/hypershift-mce-power-create-nodepool-commands.sh

```diff
@@ -273,6 +281,11 @@ done
 echo "$(date) Setup nested_kubeconfig"
 ${HYPERSHIFT_CLI_NAME} create kubeconfig --namespace=${CLUSTERS_NAMESPACE} --name=${HOSTED_CLUSTER_NAME} >${SHARED_DIR}/nested_kubeconfig
 
+if [ ${USE_GLB} == "no" ]; then
```

Comment:
I think we should not set this condition based on USE_GLB. Please think also when someone sets `USE_GLB` on regular agent conformance workflow. Either we can do it in heterogeneous related step or decide based on some other flag.
Ideally since we are pointing `*.apps` to first worker, then ingress controller should spawn on first worker.
Please modify accordingly!